pennylane/beta/devices/__init__.py

```diff
@@ -17,4 +17,3 @@
 .. currentmodule:: pennylane.beta.devices
 """
 from .default_tensor import DefaultTensor
-from .default_tensor_tf import DefaultTensorTF
```

Comment:
A heads-up here: these devices are being deprecated as of #1981. We're preparing the v0.20.0 release of PennyLane this week and will make the release early next week. By that time, these two devices will be removed in the `master` branch.